lib/tessel/access-point.js

```diff
@@ -126,7 +128,10 @@ Tessel.prototype.createAccessPoint = function(opts) {
 
     var turnAccessPointOn = () => this.simpleExec(commands.turnAccessPointOn());
 
-    var commitAndClosePromise = () => new Promise((resolve, reject) => commitAndClose(this, status, resolve, reject));
+    var commitAndClosePromise = () => {
+      log.spinner.stop();
```

Comment:
Could this be called in `commitAndClose` since that's another async action that restarts a few system processes like `wifi`, `dnsmasq`, and `dhcp`, which could take a few moments worthy of a spinner.